script/test/fixtures/multiple-compose-files/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+web:
+  image: tuna/docker-counter23
+  ports:
+    - "5000:5000"
+  links:
+    - redis
+redis:
+  image: redis:3.0
+  ports:
+    - "6379"
```

Comment:
a real world example where things get over-written will make more sense as opposed to mixing two apps.